app.go

```diff
@@ -32,7 +32,9 @@ type app struct {
 
 func newApp() *app {
 	ui := newUI()
-	nav := newNav(ui.wins[0].h)
+	// TODO: is this really the right way?
+	lastWin := ui.wins[len(ui.wins)-1]
```

Comment:
Hm, can't this be influenced by `set preview`/`set nopreview` and/or other (in)direct options (`set-ratios...`)?